src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -365,12 +382,14 @@ def _get_acr_cred(registry_name):
 
 
 def delete_container_settings(resource_group_name, name, slot=None):
+    _delete_linux_fx_version(resource_group_name, name)
     delete_app_settings(resource_group_name, name, CONTAINER_APPSETTING_NAMES, slot)
 
 
 def show_container_settings(resource_group_name, name, slot=None):
     settings = get_app_settings(resource_group_name, name, slot)
-    return _mask_creds_related_appsettings(_filter_for_container_settings(settings))
+    return "'docker-custom-image-name': {} {}".format(_get_linux_fx_version(resource_group_name, name),
```

Comment:
You should update the dictionary, rather do string format which will break the output on non json mode